package.json

```diff
@@ -23,91 +23,76 @@
-    "commitizen": "2.9.6",
     "condition-circle": "2.0.1",
-    "cross-env": "5.1.3",
-    "css-loader": "0.28.9",
-    "cssnano": "3.10.0",
+    "cross-env": "5.2.0",
+    "css-loader": "1.0.0",
     "cz-conventional-changelog": "2.1.0",
-    "enzyme": "3.3.0",
-    "enzyme-adapter-react-16": "1.1.1",
-    "enzyme-to-json": "3.3.1",
-    "eslint": "4.17.0",
-    "eslint-config-airbnb": "16.1.0",
-    "eslint-config-prettier": "2.9.0",
-    "eslint-plugin-flowtype": "2.44.0",
-    "eslint-plugin-import": "2.8.0",
-    "eslint-plugin-jsx-a11y": "6.0.3",
-    "eslint-plugin-react": "7.6.1",
+    "enzyme": "3.4.4",
+    "enzyme-adapter-react-16": "1.2.0",
+    "enzyme-to-json": "3.3.4",
+    "eslint": "5.4.0",
+    "eslint-config-airbnb": "17.1.0",
+    "eslint-config-prettier": "3.0.1",
+    "eslint-plugin-flowtype": "2.50.0",
+    "eslint-plugin-import": "2.14.0",
+    "eslint-plugin-jsx-a11y": "6.1.1",
+    "eslint-plugin-react": "7.11.1",
     "extract-text-webpack-plugin": "3.0.2",
-    "find-up": "2.1.0",
-    "flow-bin": "0.70.0",
-    "flow-copy-source": "1.3.0",
+    "find-up": "3.0.0",
+    "flow-bin": "0.79.1",
+    "flow-copy-source": "2.0.2",
     "html-loader": "0.5.5",
-    "husky": "0.14.3",
-    "ignore-loader": "0.1.2",
-    "jest": "22.3.0",
-    "lint-staged": "6.1.0",
-    "lodash": "4.17.5",
-    "markdown-loader": "2.0.2",
-    "nightwatch": "0.9.19",
-    "npm-run-all": "4.1.2",
-    "picturebook": "1.2.4",
-    "polished": "1.9.2",
     "postcss-cssnext": "3.1.0",
-    "postcss-discard-comments": "2.0.4",
-    "postcss-flexbugs-fixes": "3.3.0",
-    "postcss-for": "2.1.1",
-    "postcss-import": "11.1.0",
+    "postcss-import": "12.0.0",
     "postcss-loader": "2.1.0",
-    "postcss-nesting": "4.2.1",
-    "prettier": "1.10.2",
+    "postcss-nesting": "6.0.0",
+    "ignore-loader": "0.1.2",
+    "jest": "23.5.0",
+    "lodash": "4.17.10",
+    "markdown-loader": "3.0.0",
+    "nightwatch": "0.9.21",
+    "npm-run-all": "4.1.3",
+    "picturebook": "1.2.6",
+    "polished": "1.9.3",
+    "postcss-loader": "3.0.0",
+    "prettier": "1.14.2",
     "raw-loader": "0.5.1",
-    "react": "16.2.0",
-    "react-dom": "16.2.0",
-    "react-element-to-jsx-string": "13.1.0",
-    "react-test-renderer": "16.2.0",
-    "regenerator-runtime": "0.11.1",
-    "saucelabs": "1.4.0",
-    "selenium-standalone": "6.12.0",
-    "semantic-release": "12.4.1",
-    "storybook-addon-jsx": "5.3.0",
-    "style-loader": "0.20.2",
-    "stylelint": "8.4.0",
-    "webpack": "3.11.0",
-    "webpack-bundle-analyzer": "2.10.0"
-  },
-  "lint-staged": {
-    "*.{js,css}": [
-      "format",
-      "git add"
-    ]
+    "react": "16.4.2",
+    "react-dom": "16.4.2",
+    "react-element-to-jsx-string": "14.0.1",
+    "react-test-renderer": "16.4.2",
+    "regenerator-runtime": "0.12.1",
+    "saucelabs": "1.5.0",
+    "selenium-standalone": "6.15.1",
+    "semantic-release": "15.9.9",
+    "storybook-addon-jsx": "5.4.0",
+    "webpack": "4.16.5",
```

Comment:
updated to webpack 4. I'll check if https://github.com/gymnastjs/gymnast/issues/380 is fixed after deploy